app/src/main/java/org/schabi/newpipe/local/history/StatisticsPlaylistFragment.java

```diff
@@ -338,7 +338,7 @@ private void showStreamDialog(final StreamStatisticsEntry item) {
 
         final ArrayList<StreamDialogEntry> entries = new ArrayList<>();
 
-        if (PlayerHolder.getInstance().getType() != null) {
+        if (PlayerHolder.getInstance().isPlayerOpen()) {
             entries.add(StreamDialogEntry.enqueue);
```

Comment:
Also show it in the statistics fragment
```suggestion
            entries.add(StreamDialogEntry.enqueue);

            if (PlayerHolder.getInstance().getQueueSize() > 1) {
                entries.add(StreamDialogEntry.next);
            }
```